WooCommerce/Classes/ViewRelated/Products/Edit Product/Linked Products/Cells/NumberOfLinkedProductsTableViewCell.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+class NumberOfLinkedProductsTableViewCell: UITableViewCell {
```

Comment:
Would it make sense to declare this class as final? Or are we actually planning on subclassing it in the future?

Reply:
It makes sense, done ✅